Failed to get `maven-metadata.xml` file

I'm attempting to execute the following command:
```bash
prospero.sh install -vv --channels=dev.resteasy.channels:resteasy-6.2,org.wildfly:wildfly --dir=wildfly --fpl=org.wildfly:wildfly-galleon-pack --use-default-local-cache
```

However, the `dev.resteasy.channels:resteasy-6.2/maven-metadata.xml` cannot be found by the dependency resolution. The file does exist though, https://repo1.maven.org/maven2/dev/resteasy/channels/resteasy-6.2/maven-metadata.xml. Below is the error I'm seeing:
```
org.wildfly.channel.ArtifactTransferException: dev.resteasy.channels:resteasy-6.2/maven-metadata.xml failed to transfer from https://repo1.maven.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution will not be reattempted until the update interval of central has elapsed or updates are forced. Original error: Could not transfer metadata dev.resteasy.channels:resteasy-6.2/maven-metadata.xml from/to central (https://repo1.maven.org/maven2): Cannot access https://repo1.maven.org/maven2 with type  using the available connector factories: BasicRepositoryConnectorFactory
	at org.jboss.prospero//org.wildfly.channel.maven.RetryHandler$RetryCounter.addRetry(RetryHandler.java:144)
	at org.jboss.prospero//org.wildfly.channel.maven.RetryHandler.attemptResolveMetadata(RetryHandler.java:109)
	at org.jboss.prospero//org.wildfly.channel.maven.VersionResolverFactory$MavenResolverImpl.getAllVersions(VersionResolverFactory.java:153)
	at org.jboss.prospero//org.wildfly.channel.maven.VersionResolverFactory$MavenResolverImpl.resolveChannelMetadata(VersionResolverFactory.java:238)
	at org.jboss.prospero//org.wildfly.channel.maven.VersionResolverFactory.resolveChannels(VersionResolverFactory.java:340)
	at org.jboss.prospero//org.wildfly.prospero.api.ProvisioningDefinition.resolveChannels(ProvisioningDefinition.java:177)
	at org.jboss.prospero//org.wildfly.prospero.cli.commands.AbstractInstallCommand.resolveChannels(AbstractInstallCommand.java:93)
	at org.jboss.prospero//org.wildfly.prospero.cli.commands.InstallCommand.call(InstallCommand.java:149)
	at org.jboss.prospero//org.wildfly.prospero.cli.commands.InstallCommand.call(InstallCommand.java:50)
	at org.jboss.prospero//picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at org.jboss.prospero//picocli.CommandLine.access$1500(CommandLine.java:148)
	at org.jboss.prospero//picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at org.jboss.prospero//picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at org.jboss.prospero//picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at org.jboss.prospero//picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at org.jboss.prospero//picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at org.jboss.prospero//picocli.CommandLine.execute(CommandLine.java:2170)
	at org.jboss.prospero//org.wildfly.prospero.cli.CliMain.execute(CliMain.java:132)
	at org.jboss.prospero//org.wildfly.prospero.cli.CliMain.main(CliMain.java:60)
	at org.jboss.modules.Module.run(Module.java:352)
	at org.jboss.modules.Module.run(Module.java:320)
	at org.jboss.modules.Main.main(Main.java:607)
Caused by: org.eclipse.aether.transfer.MetadataTransferException: dev.resteasy.channels:resteasy-6.2/maven-metadata.xml failed to transfer from https://repo1.maven.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution will not be reattempted until the update interval of central has elapsed or updates are forced. Original error: Could not transfer metadata dev.resteasy.channels:resteasy-6.2/maven-metadata.xml from/to central (https://repo1.maven.org/maven2): Cannot access https://repo1.maven.org/maven2 with type  using the available connector factories: BasicRepositoryConnectorFactory
	at org.jboss.prospero//org.eclipse.aether.internal.impl.DefaultUpdateCheckManager.newException(DefaultUpdateCheckManager.java:331)
	at org.jboss.prospero//org.eclipse.aether.internal.impl.DefaultUpdateCheckManager.checkMetadata(DefaultUpdateCheckManager.java:293)
	at org.jboss.prospero//org.eclipse.aether.internal.impl.DefaultMetadataResolver.resolve(DefaultMetadataResolver.java:303)
	at org.jboss.prospero//org.eclipse.aether.internal.impl.DefaultMetadataResolver.resolveMetadata(DefaultMetadataResolver.java:180)
	at org.jboss.prospero//org.apache.maven.repository.internal.DefaultVersionRangeResolver.getVersions(DefaultVersionRangeResolver.java:198)
	at org.jboss.prospero//org.apache.maven.repository.internal.DefaultVersionRangeResolver.resolveVersionRange(DefaultVersionRangeResolver.java:148)
	at org.jboss.prospero//org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveVersionRange(DefaultRepositorySystem.java:260)
	at org.jboss.prospero//org.wildfly.channel.maven.VersionResolverFactory$MavenResolverImpl.lambda$getAllVersions$0(VersionResolverFactory.java:155)
	at org.jboss.prospero//org.wildfly.channel.maven.RetryHandler.attemptResolveMetadata(RetryHandler.java:96)
	... 20 more
```

The `~/.m2/repository/dev/resteasy/channels/resteasy-6.2/resolver-status.properties` contains the following:
```properties
#NOTE: This is a Maven Resolver internal implementation file, its format can be changed without prior notice.
#Fri Jun 21 10:15:38 PDT 2024
maven-metadata-central.xml.error=Could not transfer metadata dev.resteasy.channels\:resteasy-6.2/maven-metadata.xml from/to central (https\://repo1.maven.org/maven2)\: Cannot access https\://repo1.maven.org/maven2 with type  using the available connector factories\: BasicRepositoryConnectorFactory
maven-metadata-central.xml/@-central-https\://repo1.maven.org/maven2/.lastUpdated=1718990095779
```

I'm not too sure what the issue could be.

--- a/prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java
+++ b/prospero-common/src/main/java/org/wildfly/prospero/api/ProvisioningDefinition.java
@@ -139,7 +139,7 @@ public class ProvisioningDefinition {
 
     private List<RemoteRepository> channelResolutionRepositories() {
         if (overrideRepositories.isEmpty()) {
-            return List.of(new RemoteRepository.Builder("central", null, "https://repo1.maven.org/maven2").build());
+            return List.of(new RemoteRepository.Builder("central", "default", "https://repo1.maven.org/maven2").build());
         } else {
             return overrideRepositories.stream()
                     .map(RepositoryUtils::toRemoteRepository)

